### PR TITLE
Add speaker notes for the first few slides

### PR DESCRIPTION
**Pull request by mgeisler**

### CODE REVIEW

**Comment by mgeisler** [comment]
@zdimension and @sharnoff, this rephrases the footnote about memory leaks a bit... I hope you feel I kept the important points from #28.

**Comment by sharnoff** [comment]
Looks great!

File: src/welcome.md [hunk 1]
```diff
@@ -42,3 +42,12 @@ explain or contrast the Rust approach.
 
 If you know how to program in a dynamically typed language such as Python or
 JavaScript, then you will be able to follow along just fine too.
+
+<details>
+
+This is an example of a _speaker note_. We will use these to add
+additional information to the slides. This could be key points which
+the instructor should cover as well as explanations to typical
```
Comment (qwandor) [inline src/welcome.md:50]: "explanations to" doesn't really make sense, maybe "answers to" or "explanations in response to".

File: src/why-rust.md [hunk 2]
```diff
@@ -5,3 +5,20 @@ Some unique selling points of Rust:
...
+
+* Experience with C or C++: Rust eliminates a whole class of _runtime
+  errors_ via the borrow checker. You get performance like in C and
+  C++, but you don't have the memory unsafety issues. In addition, you
+  get a modern language with constructs like pattern matching and
+  built-in dependency management.
+
+* Experience with Java, Python, JavaSript...: You get the same memory
+  safety as in those languages, plus a similar high-level language
+  feeling. In addition you get performance like in C and C++.
```
Comment (qwandor) [inline src/why-rust.md:22]: Not just performance but predictability (no surprise pauses for the GC), and potentially lower-level access to hardware.

File: src/why-rust/modern.md [hunk 3]
```diff
@@ -13,3 +13,41 @@ Rust is built with all the experience gained in the last 40 years.
...
+
+  The reasoning behind this is that functionality in the standard
+  library cannot go away, so it has to be very stable. For the
+  examples above, the Rust community is still working on finding the
+  best solution --- and perhaps there isn't a single "best solution"
+  for some of these things.
+
+  Rust comes with a built-in package manager in the form of Cargo and
+  this makes it trivial to download and compile third-party crates.
+  A consequence of this is that  the standard library can be smaller.
```
Comment (qwandor) [inline src/why-rust/modern.md:43]: Extra space between 'that' and 'the'.

**Review by qwandor — APPROVED** [review]

File: src/welcome.md (hunk 1 above)
Comment (mgeisler) [reply src/welcome.md:50]: Thanks!

**Pull request merged**